DependencyInjection/Configuration.php

```diff
@@ -41,7 +41,7 @@ public function getConfigTreeBuilder() : TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('dir_name')->defaultValue('%kernel.root_dir%/DoctrineMigrations')->cannotBeEmpty()->end()
+                ->scalarNode('dir_name')->defaultValue('%doctrine.migrations.dir%')->cannotBeEmpty()->end()
```

Comment:
to stick with the naming of the other parameters I would name this one `%doctrine_migrations.default_dir_name%`